### ISSUE TITLE
binding for getter putting expression in output as literal

### ISSUE THREAD

**Issue by 4deeptech** [jdanyow/aurelia-computed#10]
I copied the skeleton app script and html files into a new project so that I could learn how to get everything(aurelia and tools) set up from scratch.  Installing node, npm, jspm, gulp, etc.  I managed to get the skeleton working all except for one thing.  The fullName get call when bound in the html actually returns the literal '' in upper case.

HTML binding expression

<p class="help-block">${fullName | upper}</p>

Output:
${THIS.FIRSTNAME} ${THIS.LASTNAME}

Instead of outputting: 
JOHN DOE

If I change the html to:

<p class="help-block">${firstName | upper} ${lastName | upper}</p>

It properly binds and outputs: JOHN DOE

I'm not sure what is happening. No errors in the console and it says it loaded and configured the plugin. Not sure where to look for issue.

**Comment by jdanyow** [comment]
Can you share your `get fullName()` property?  It looks like you might not be using back-tics ` to enclose the string.

**Comment by 4deeptech** [comment]
Jeremy,  I bet your are 100% correct.  I haven't tried it yet as I am not where I can run it on that machine at the moment. However, I do remember seeing those quotes flagged by visual studio when I was trying to get Aurelia bootstrapped and working and managed to swap them out not knowing they were valid. Also, I did not realize that ES6 uses those for template strings.  I found info on the spec. Chock it up to ignorance of ES6. I have a lot to learn to catch up.  I will let you know if that is indeed the issue.
